webpack.config.js

```diff
@@ -42,10 +41,8 @@ module.exports = function(env) {
 		}
 
 		config.plugins.push(
-			new webpack.optimize.UglifyJsPlugin({
-				include: /\.min\.js$/,
-				minimize: true
-			}), new webpack.BannerPlugin(banner.pkgd)
+			new UglifyJSPlugin(uglifyConfig),
+			new webpack.BannerPlugin(banner.pkgd)
```

Comment:
```js
new webpack.BannerPlugin(banner.common + banner.pkgd)
```
And correct the top comment banner when referring single project, it should be singular rather than plural.
```js
 * @egjs/flicking projects are licensed under the MIT <https://naver.github.io/egjs/license.txt> license
```